Cartfile

```diff
@@ -0,0 +1 @@
+github "krzyzanowskim/CryptoSwift" == 0.10.0
```

Comment:
is it a needed dependency? Why import the whole cryptoSwift just for the MD5 calculation?

Reply:
It depends. 1) Do you want to take parts of your binary dependencies and import them manually? 2) Do you want to just have a list of dependencies in a Cartfile and just let Carthage handle it? It's your call.

Reply:
We need this dependency just to make the MD5 calculation, and there are other ways that do not involve to depend on other libraries to do that.

Reply:
hi @rob-nash just a request from another user, I personally would appreciate it if you don't swear on here.  Thanks.  (I swear on other forums all the time - I just don't like swearing on here ;)  )

BTW I have never in the slightest found gringoireDM to be rude, maybe you read something wrong. 

Regarding this technical issue,I agree with this "It mainly aims to move users from this repo to a private repo to see examples and demo. For me it's a no go."

Cheers all